standalone-kafka/docker-compose.yml

```diff
@@ -1,44 +1,47 @@
 ---
-version: '3'
-
 services:
-  zookeeper:
-    image: ${DOCKER_REGISTRY:-docker.io}/confluentinc/cp-zookeeper:${ZOOKEEPER_VERSION:-latest}
-    container_name: zookeeper
-    hostname: zookeeper
-    environment:
-      TZ: ${TIMEZONE:-America/New_York}
-      ZOOKEEPER_CLIENT_PORT: 2181
-      ZOOKEEPER_TICK_TIME: 2000
-
   broker:
-    image: ${DOCKER_REGISTRY:-docker.io}/confluentinc/cp-kafka:${KAFKA_VERSION:-latest}
-    container_name: broker
+    container_name: kafka
+    image: quay.io/strimzi/kafka:latest-kafka-3.7.0
     hostname: broker
-    depends_on:
-      - zookeeper
+    command:
+      [
+        "sh",
+        "-c",
+        "export CLUSTER_ID=$$(bin/kafka-storage.sh random-uuid) && bin/kafka-storage.sh format -t $$CLUSTER_ID -c config/kraft/server.properties && bin/kafka-server-start.sh config/kraft/server.properties --override advertised.listeners=$${KAFKA_ADVERTISED_LISTENERS} --override listener.security.protocol.map=$${KAFKA_LISTENER_SECURITY_PROTOCOL_MAP} --override listeners=$${KAFKA_LISTENERS}",
+      ]
+    ports:
+      - "9092:9092"
     environment:
-      TZ: ${TIMEZONE:-America/New_York}
-      KAFKA_BROKER_ID: 1
-      KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://localhost:9092
+      TZ: Europe/Berlin
```

Comment:
Should probably leave as `${TIMEZONE:-America/New_York}`

Reply:
Good catch. Thank you!